tornjak-frontend/src/components/dashboard/entries-dashboard-table.js

```diff
@@ -39,16 +39,19 @@ class EntriesDashBoardTable extends React.Component {
     }
 
     entriesList = this.props.globalEntries.globalEntriesList.map(currentEntry => {
-      return this.TornjakHelper.workloadEntry(currentEntry, this.props.globalAgents.globalAgentsWorkLoadAttestorInfo);
+      return this.TornjakHelper.workloadEntry(currentEntry, this.props.globalAgents.globalAgentsWorkLoadAttestorInfo, this.props.globalAgents.globalAgentsList, this.props.globalEntries.globalEntriesList);
     })
 
     // For details page filtering data
     if (filterByCluster === undefined && filterByAgentId === undefined) {
```

Comment:
```
if (!filterByCluster && !filterByAgentId) {
```

Reply:
File not of concern on this PR. Will be addressed in the next consecutive PR.